### DISCUSSION TITLE
It seems to me that this project must be licensed under GPL

### DISCUSSION THREAD

**Discussion by schrieveslaach** [nvim-java/nvim-java#133] [General]
I read following in the projects README

> [nvim-jdtls](https://github.com/mfussenegger/nvim-jdtls) is a plugin that follows "Keep it simple, stupid!" approach. If you love customizing things by yourself, then give nvim-jdtls a try. I may or may not have copied some code 😉 Beauty of Open source!

I read that that code has been copied from nvim-jdtls to nvim-java. [nvim-jdtls is licensed under GPLv3](https://github.com/mfussenegger/nvim-jdtls?tab=GPL-3.0-1-ov-file#readme) and therefore, I think nvim-java has to be licensed under GPLv3 too: https://www.patrick-wied.at/blog/how-to-correctly-use-code-you-didnt-write

CC: @mfussenegger

**Comment by s1n7ax** [comment]
@schrieveslaach Damn. I never knew that about GPL-3. But, there are not literal copies. I have used it to understand how LSP setting up process working. I have used `vscode-java`, `vscode-java-debug`, `vscode-java-test` as JDTLS documentation since there is no documentation for LSP APIs. If you take a look at the code, it's entirely different. (Not saying there are no similarities BTW)
